pkg/p2p/testbed/cluster_test.go

```diff
@@ -0,0 +1,153 @@
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT License was not distributed with this
+// file, you can obtain one at https://opensource.org/licenses/MIT.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+//go:build testbed
+// +build testbed
+
+package testbed
+
+import (
+	"context"
+	"encoding/hex"
+	"io"
+	"os/exec"
+	"strconv"
+	"sync"
+
+	"github.com/dusk-network/dusk-protobuf/autogen/go/rusk"
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
```

Comment:
Any reason for the `logrus` import to be both named and unnamed?

Reply:
typo